docs/slack-setup.md

```diff
@@ -0,0 +1,28 @@
+# Slack App Setup
```

Comment:
I'd maybe just put this bit in the README, then I can expand on it to create a local setup section